src/zenml/zen_server/routers/service_connectors_endpoints.py

```diff
@@ -393,3 +401,53 @@ def get_service_connector_type(
         The requested service connector type.
     """
     return zen_store().get_service_connector_type(connector_type)
+
+
+@router.post(
+    SERVICE_CONNECTOR_FULL_STACK,
+    response_model=ServiceConnectorResourcesInfo,
```

Comment:
```suggestion
```

This is already defined by the return type and having it duplicate lead to some issues before.